.github/workflows/php.yml

```
@@ -16,12 +16,12 @@ jobs:
   cs:
     uses: bedita/github-workflows/.github/workflows/php-cs.yml@v1
     with:
-      php_versions: '["8.2"]'
+      php_versions: '["8.2", "8.3"]'
```

It is sufficient to run CS and Stan against one version of PHP only…

```suggestion
      php_versions: '["8.3"]'
```